pkg/local_object_storage/engine/inhume_test.go

```diff
@@ -153,4 +154,21 @@ func TestStorageEngine_Inhume(t *testing.T) {
 		_, err = wrongShard.Get(getPrm)
 		require.ErrorAs(t, err, new(apistatus.ObjectNotFound))
 	})
+
+	t.Run("inhuming object twice", func(t *testing.T) {
```

Comment:
is it related to the 404 change? i see that it helped to find the reason, but afaik it does not assert the introduced fix

Reply:
> is it related to the 404 change

it is the situation that led to a confusing log on our machines. since it is not an error and all the problems are only about logs, i decided to add it just to be sure it is not a problem (but it really looked like a problem if you do not know this test passes and if https://github.com/nspcc-dev/neofs-node/issues/2956 is not in your mind)